src/core/mac/mac.hpp

```diff
@@ -870,6 +857,12 @@ class Mac : public InstanceLocator, private NonCopyable
 #if OPENTHREAD_CONFIG_MAC_CSL_TRANSMITTER_ENABLE
     void ProcessCsl(const RxFrame &aFrame, const Address &aSrcAddr);
 #endif
+#if OPENTHREAD_CONFIG_MAC_CSL_RECEIVER_ENABLE
+    void UpdateCslParameters(void);
+    void UpdateCslState(void);
+    void SetCslParameters(uint16_t aCslPeriod);
+    bool IsCslFeatureEnabled(void) const { return mCslPeriod > 0; }
```

Comment:
I think this method is confusing. We have `IsCslEnabled` and `IsCslFeatureEnabled`. Considering this method is used only once, I suggest just write `mCslPeriod > 0` in the condition. Thoughts?

Reply:
I will change it.